microsoft/testsuites/dpdk/dpdksuite.py

```diff
@@ -81,28 +68,79 @@ def _execute_expect_zero(self, node: Node, cmd: str) -> str:
         result.assert_exit_code()
         return result.stdout
 
-    def _hugepages_init(self, node: Node) -> None:
-        mount = node.tools[Mount]
-        mount.mount(disk_name="nodev", point="/mnt/huge", type="hugetlbfs")
-        mount.mount(
-            disk_name="nodev",
-            point="/mnt/huge-1G",
-            type="hugetlbfs",
-            options="pagesize=1G",
-        )
+    @TestCaseMetadata(
+        description="""
+           Tests a more realistic sender/forwarder/receiver setup.
+        """,
+        priority=1,
```

Comment:
Does it need to be P1? The v2 priority should plus 1 into v3. For example, P1 in v2 should be p2 in v3.